### ISSUE TITLE
Add support for "merged-channel folders"

### ISSUE THREAD

**Issue by mintsoft** [mintsoft/kodi.plugin.ytchannels#15]
Based on this use-case: https://forum.kodi.tv/showthread.php?tid=45246&pid=2983160#pid2983160

Suggestion is to create a folder type that when channels are added to, the user is presented with all the videos for all the channels sorted by release and interlaced with each other.

I can imagine that users might use it to merge a first and second channel for a given youtuber into one "virtual channel" or a pipeline of all their subscriptions together. 

No idea what to call them, I keep coming up with names like "mixtape" or "playlist channels" or "virtual channels" or "mosiac channels"; definitely needs a better name.

**Comment by yeahme49** [comment]
Trying to determine the best way to do this. Grab X number of videos from each channel, then sort them all so they are intermixed by date? Or grab X number of videos from each channel and then list them with Channel 1's videos listed first, then Channel 2, etc.

I like the 1st way better, but then you run into issues where if channel 1's videos are all from the past week but channel 2's videos are from the past month. You are listing week old videos then month old videos and when you go to the next page, you might have videos that are all of a sudden 2 weeks old but they got listed after videos that are a month old.

Any thoughts?

**Comment by mintsoft** [comment]
Yeah I think the first way is better. Like I'd want it as a time-feed rather than a list of channel X then a list of channel Y.
